Toolkit/AppDeployToolkit/AppDeployToolkitMain.ps1

```diff
@@ -6480,7 +6480,7 @@ https://psappdeploytoolkit.com
                     Get-ItemProperty -LiteralPath $_.PSPath -ErrorAction 'Stop' | Where-Object { ($_.ProfileImagePath) } |
                         Select-Object @{ Label = 'NTAccount'; Expression = { $(ConvertTo-NTAccountOrSID -SID $_.PSChildName).Value } }, @{ Label = 'SID'; Expression = { $_.PSChildName } }, @{ Label = 'ProfilePath'; Expression = { $_.ProfileImagePath } }
                     } |
-                    Where-Object { $_.NTAccount } # This removes the "defaultuser0" account, which is a Windows 10 bug
+                    Where-Object { $_.NTAccount -notmatch 'defaultuser0$'} # This excludes the "defaultuser0" account used by CloudExperienceHostBroker.exe
```

Comment:
Same here also.

Reply:
Just a note, this functionality is used by others. We utilise PSADT to enforce a range of configurations during Autopilot that arent available via traditional CSP / Settings Catalogue or they might not fit for purpose in their current state via Intune's native featureset.

Eg, a subset of security baseline ADMX based configurations target HKU instead of HKLM, Intune does not enforce the configurations into defaultuser0 profiles (they dont get deleted after OOBE if the logged in users are all standard, raised this with MS recently & have a change request in for it).

We utilise Qualys for our Security Baseline compliance reporting which reports that defaultuser0 is missing the configurations. Utilising PSADT to enforce the configurations to all profiles is a path of least resistance for us as it is quite difficult getting Qualys to modified the control evaluation logic for a number of configurations.

TLDR, this would introduce a breaking change for our (fringe) usage, of course we shouldnt need to leverage PSADT in the first place in the perfect world....